spec/requests/waste_exemptions_engine/applicant_email_forms_spec.rb

```diff
@@ -16,5 +16,25 @@ module WasteExemptionsEngine
         ]
       end
     end
+
+    context "when editing an existing registration" do
+      let(:edit_applicant_email_form) { build(:edit_applicant_email_form) }
+
+      it "prefils applicant email information" do
```

Comment:
```suggestion
      it "pre-fills applicant email information" do
```

Reply:
Oh!!!! Really??? That's the way you spell that??? Shut!! I really didn't know 😆 🤣